Automattic/jetpack
# Update PHP 7.4 to 8.0 in our tools

#### Changes proposed in this Pull Request:

* Updates PHP version constant in `.github/versions.sh` from `7.4` to `8.0`. This constant value is used in various monorepo tools. [Active support](https://www.php.net/supported-versions.php) for 7.4 is elapsed.
* Removes `php${PHP_VERSION}-json` from the Dockerfile. _"As of PHP 8.0.0, the JSON extension is a core PHP extension, so it is always enabled."_ [source](https://www.php.net/manual/en/json.installation.php)

#### Jetpack product discussion

Internal: p9dueE-3Vv-p2#comment-6799

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- Make sure your Jetpack Docker container is stopped.
- Pull down changes in this PR.
- Run `jetpack docker build-image`. Check to see if there are any errors reported during the build process.
- Next run `jetpack docker up` to start your Jetpack container.
- Once the Jetpack container is up run `jetpack docker sh`. When you see the Docker instance shell run `php -v` to see the updated PHP version in use, 8.0.z (it would have been 7.4.z previously).

After this PR is merged, we can also check that the `Build Docker` workflow published an updated image https://hub.docker.com/r/automattic/jetpack-wordpress-dev
